js/test/unit/table-tests.ts

```diff
@@ -234,7 +234,7 @@ describe(`Table`, () => {
             expect(f32.name).toBe('f32Renamed');
             expect(i32.length).toBe(i32s.length);
             expect(f32.length).toBe(i32s.length); // new length should be the same as the longest sibling
-            expect(i32.nullable).toBe(false);
+            expect(i32.nullable).toBe(true);
```

Comment:
Did all of these have to change because of the change in `_selectFieldArgs`, or is there something else going on here?